app/kubemci/cmd/create.go

```diff
@@ -243,20 +261,30 @@ func unmarshall(filename string, ing *v1beta1.Ingress) error {
 	return nil
 }
 
-func unmarshallAndApplyDefaults(filename string, ing *v1beta1.Ingress) error {
+func addAnnotation(ing *v1beta1.Ingress, key, val string) {
```

Comment:
optional: Will be great if you can add a TODO to move this to ingress-gce/pkg/annotations